feature/converter/converter.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+
+class Converter(object):
+
+    def __init__(self, first_file_name, second_file_name):
+        self.first_file_name = first_file_name
```

Comment:
No need to change anything, it is your choice, just an advice: I would prefer these variables called
`self.origin_file_name and self.destination_file_name` or `self.origin_file_path and self.destination_file_path`

Just because we have an origin file in one format and destination file in another format. Also, if our origin file isn't in directory where the script is, then we would need to type in a path to file, rather than it's name.